src/common/primitive_attr_quant.hpp

```diff
@@ -80,6 +80,8 @@ struct quant_entry_t : public c_compatible {
 
     int get_mask() const { return mask_; }
     data_type_t get_data_type() const { return data_type_; }
+    int get_group_ndims() const { return group_ndims_; }
+    const dims_t &get_group_dims() const { return group_dims_; }
```

Comment:
Discussed offline. The proposal is to introduce a `deserialize` method to quant_entry, scales and zero-points instead of exposing internals.

Reply:
Updated to use common API.